lessons/Data-Science/variability/langs/en-us/pages/animal-weight-variability.adoc

```diff
@@ -0,0 +1,71 @@
+= Variability of Animals' Weights
+
+@link-instructions{
+The folks at the shelter want to approximate the amount of food they need to purchase for the coming month. They know there is a relationship between an animal's weight and how much it eats, so they are discussing the distribution of animals' weights. Help them out by responding to the prompts below. For question 7, you'll need to use the @starter-file{expanded-animals-sub-pops}.
+}
+
+@n Imagine a _typical_ animal from each of these four species, and estimate its weight. (To get you started, we've provided a weight estimate for a typical dog.) @hspace{2em} dog: @fitb{7em}{55 lbs} @hspace{2em} rabbit: @fitb{7em}{}  @hspace{2em} cat: @fitb{7em}{} @hspace{2em} tarantula: @fitb{7em}{}
```

Comment:
@retabak this page feels like a heavy lift of abstract thinking. In my experience, students have no frame of reference for estimating weight. I wonder if we should provide them with the weights of some things that they would be able to imagine these animals in relation to. (hamster, sheep, dalmation, chihuahua?)